docs/command-line/promtool.md

```diff
@@ -431,9 +431,10 @@ Unit tests for rules.
 
 ###### Flags
 
-| Flag | Description |
-| --- | --- |
-| <code class="text-nowrap">--run</code> | If set, will only run test groups whose names match the regular expression. Can be specified multiple times. |
+| Flag | Description | Default |
+| --- | --- | --- |
+| <code class="text-nowrap">--run</code> | If set, will only run test groups whose names match the regular expression. Can be specified multiple times. |  |
+| <code class="text-nowrap">--diff</code> | Print colored differential output between expected & received output | `false` |
```

Comment:
Very tiny thing

```suggestion
| <code class="text-nowrap">--diff</code> | Print colored differential output between expected & received output. | `false` |
```

Reply:
+1, but that change needs to be in the source code, not the autogenerated docs.

While here I think we should mark this as experimental for now. This is great, but I think eventually `--diff` should just be the default.

Reply:
Good idea. Just specifying that in the flag description should do.

Reply:
I've made the very tiny upgrade @LeviHarrison!